zmsadmin/js/page/availabilityDay/layouts/tableBody.js

```
@@ -1,42 +1,135 @@
+            : <i className="fas fa-sort-down" aria-hidden="true"></i>;
+    };
+
+    const sortData = (data) => {
+        if (!sortColumn) return data;
+
+        return [...data].sort((a, b) => {
+            const multiplier = sortDirection === 'asc' ? 1 : -1;
+            
+            switch (sortColumn) {
+                case 'weekday':
+                    const aWeekDay = Object.keys(a.weekday).find(key => parseInt(a.weekday[key], 10) > 0) || '';
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)